en/intro-concepts.md

```diff
@@ -13,18 +13,23 @@ MAAS, not to mention the rest of this documentation.
 ## Nodes
 
 A *node* is a general term that refers to multiple, more specific objects.
-Basically, it is a networked object that is known to MAAS.
+Nodes are managed by MAAS through a life cycle, from enlistment into MAAS,
```

Comment:
Enlistment is the auto-add feature. It is also possible to manually add (so no enlistment).